app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -25,6 +25,7 @@
     private static JestDroidClient client;
     private static final String groupIndex = "cmput301w17t11";
     private static final String typeMood = "mood";
+    private static final String typeUser = "user";
```

Comment:
We can now push user accounts to Elasticsearch as well as Moods. This is vital for our login screen, and for participant sorting in general.